build-system/tasks/build.js

```diff
@@ -61,6 +61,7 @@ async function doBuild(extraArgs = {}) {
   const options = {
     fortesting: extraArgs.fortesting || argv.fortesting,
     minify: false,
+    npm: argv.npm,
```

Comment:
Does the `npm` flag get read somewhere? The `options.npm` in `build-system/tasks/extension-helpers.js` is reading the `ExtensionOption` definition, not this boolean.

Reply:
npm flag gets read here eventually - https://github.com/ampproject/amphtml/blob/main/build-system/tasks/extension-helpers.js#L494

I missed this definition. So`Map<string, NpmBinaries>` the intended type? So should I pass `{extension: {preact: true, react: true}}` ?

Reply:
Oh nevermind, I see it coming from `bundles.config.extensions.json`. One sec...

Reply:
Guess I didn't have to change anything!